RoeiJeRot.Test/LoginTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using RoeiJeRot.Database.Database;
+using RoeiJeRot.Logic;
+using RoeiJeRot.Logic.Services;
+
+namespace RoeiJeRot.Test
+{
+    class TestUserService : IUserService
+    {
+        public List<User> GetUsers()
+        {
+            List<User> list = new List<User>();
+            list.Add(new User()
+            {
+                FirstName = "Paul",
```

Comment:
Andere data dan de username/password kan je weg laten. Is beter omdat dat in de toekomst alleen maar problemen kan veroorzaken als je aanpassingen updates/maakt, want dan moet je dit ook updaten.